src/AltinnCore/Common/Services/Implementation/InstanceSILocalDev.cs

```diff
@@ -161,7 +171,6 @@ public async Task<Instance> UpdateInstance<T>(T dataToSerialize, string applicat
         /// <param name="applicationId">application id</param>
         /// <param name="applicationOwnerId">application owner id</param>
         /// <param name="instanceOwnerId">instance owner id</param>
```

Comment:
Why is this removed if its still a param:
/// <param name="instanceId">instance id</param>

Reply:
added